lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -8,38 +8,23 @@ module Fastlane
 
   module Helper
     class GithubHelper
-      def self.github_token!
-        token = [
-          'GHHELPER_ACCESS', # For historical reasons / backward compatibility
-          'GITHUB_TOKEN',    # Used by the `gh` CLI tool
-        ].map { |key| ENV[key] }
-                .compact
-                .first
-
-        token || UI.user_error!('Please provide a GitHub authentication token via the `GITHUB_TOKEN` environment variable')
-      end
```

Comment:
Yay! Bye bye `GithubHelper#github_token!` method and the need for the helper to know or look at the env vars  🎉